### PR TITLE
update benchmark blost with kinvolks's update

### PR DESCRIPTION
**Pull request by wmorgan**
Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/linkerd-benchmarks.md [hunk 1]
```diff
@@ -72,6 +79,11 @@ worse than Linkerd's p99. Its p999 was well over a second, compared to
 Linkerd's 679ms, and its worst case was a full 5s of latency, 2.5x what was
 measured with Linkerd.
 
+(**Update**: Kinvolks's re-tuned Istio benchmarks dropped Istio's p99 from 100x
+that of Linkerd's to 26x and 59x that of Linkerd's across two runs. It also
+dropped Istio's p999 to just under a second, thought still *double* Linkerd's
```
Comment (rmars) [inline linkerd.io/content/blog/linkerd-benchmarks.md]: s/thought/though

File: linkerd.io/content/blog/linkerd-benchmarks.md [hunk 2]
```diff
@@ -80,6 +92,10 @@ almost 6 minutes. Even Istio's p50 (median) latency was an unacceptable 17.6
 seconds. In short, Istio was not able to perform effectively in Kinvolk's
 600rps condition.
 
+(**Update**: Kinvolks's re-tuned Istio showed similar performance in the 600rps
```
Comment (klingerf) [inline linkerd.io/content/blog/linkerd-benchmarks.md]: Missing word here -- "Kinvolks's re-tuned Istio _benchmarks_ showed..."

Comment (rmars) [inline linkerd.io/content/blog/linkerd-benchmarks.md]: I think there are too many s-es here? s/Kinvolks's/Kinvolk's

File: linkerd.io/content/blog/linkerd-benchmarks.md [hunk 3]
```diff
@@ -130,6 +146,11 @@ utilization was higher than Istio's in the 600rps condition, though it's hard
 to know how real this result was because Istio was not able to fully perform in
 this condition._**
 
+(**Update**: Kinvolks's re-tuned Istio benchmarks showed a "massive increase in
```
Comment (rmars) [inline linkerd.io/content/blog/linkerd-benchmarks.md]: same here, re s/Kinvolks's/Kinvolk's

File: linkerd.io/content/blog/linkerd-benchmarks.md [hunk 4]
```diff
@@ -130,6 +146,11 @@ utilization was higher than Istio's in the 600rps condition, though it's hard
...
+CPU usage of Isto's proxy sidecar". While no numbers were reported, from this
```
Comment (klingerf) [inline linkerd.io/content/blog/linkerd-benchmarks.md]: "Isto's" => "Istio's"

**Pull request merged**